### PR TITLE
Update build documentation

### PR DESCRIPTION
**Pull request by braydonf**

### CODE REVIEW

**Comment by braydonf** [comment]
I ran into this issue https://github.com/trezor/blockbook/issues/483, unclear if related.

**Comment by martinboehm** [comment]
Hi. Is your Blockbook running fine with RocksDB v6.8.1 or even higher?

**Comment by braydonf** [comment]
The initial sync worked, I'll do some additional testing.

File: docs/build.md [hunk 1]
```diff
@@ -244,7 +254,7 @@ Blockbook's blockchain configuration from our coin definition files.
 
 Example for Bitcoin:
 ```
-contrib/scripts/build-blockchaincfg.sh
```
Comment (IKE471) [inline docs/build.md:247]: shows red

File: docs/build.md [hunk 2]
```diff
@@ -211,18 +211,28 @@ sudo apt-get update && sudo apt-get install -y \
...
 cd rocksdb
+git checkout v6.8.1
 CFLAGS=-fPIC CXXFLAGS=-fPIC make release
 ```
 
 Setup variables for gorocksdb: https://github.com/tecbot/gorocksdb
 
 ```
 export CGO_CFLAGS="-I/path/to/rocksdb/include"
-export CGO_LDFLAGS="-L/path/to/rocksdb -lrocksdb -lstdc++ -lm -lz -lbz2 -lsnappy -llz4"
```
Comment (IKE471) [inline docs/build.md:221]: red

File: docs/build.md [hunk 3]
```diff
@@ -194,7 +194,7 @@ like macOS or Windows, please read instructions specific for each project.
 Setup go environment:
 
 ```
-wget https://dl.google.com/go/go1.14.2.linux-amd64.tar.tz && tar xf go1.14.2.linux-amd64.tar.gz
```
Comment (IKE471) [inline docs/build.md:197]: shows red

**Review by IKE471 — APPROVED** [review]

**Review by IKE471 — APPROVED** [review]

File: .gitignore [hunk 4]
```diff
@@ -8,6 +8,7 @@ debug*
 docker/blockbook
 build/pkg-defs
 build/blockbook
+build/blockchaincfg.json
```
Comment (IKE471) [inline .gitignore:11]: its green

File: docs/build.md [hunk 5]
```diff
@@ -244,7 +254,7 @@ Blockbook's blockchain configuration from our coin definition files.
 
 Example for Bitcoin:
 ```
-contrib/scripts/build-blockchaincfg.sh
+./contrib/scripts/build-blockchaincfg.sh <coin>
```
Comment (IKE471) [inline docs/build.md:257]: green

File: docs/build.md [hunk 6]
```diff
@@ -211,18 +211,28 @@ sudo apt-get update && sudo apt-get install -y \
...
 
+```
+git clone https://github.com/zeromq/libzmq
+cd libzmq
+./autogen.sh
+./configure
+make
+sudo make install
+```
+
```
Comment (IKE471) [inline docs/build.md:235]: 227 to 235, all green

File: docs/build.md [hunk 7]
```diff
@@ -211,18 +211,28 @@ sudo apt-get update && sudo apt-get install -y \
...
+git checkout v6.8.1
 CFLAGS=-fPIC CXXFLAGS=-fPIC make release
 ```
 
 Setup variables for gorocksdb: https://github.com/tecbot/gorocksdb
 
 ```
 export CGO_CFLAGS="-I/path/to/rocksdb/include"
-export CGO_LDFLAGS="-L/path/to/rocksdb -lrocksdb -lstdc++ -lm -lz -lbz2 -lsnappy -llz4"
+export CGO_LDFLAGS="-L/path/to/rocksdb -ldl -lrocksdb -lstdc++ -lm -lz -lbz2 -lsnappy -llz4"
```
Comment (IKE471) [inline docs/build.md:222]: looks good

File: docs/build.md [hunk 8]
```diff
@@ -194,7 +194,7 @@ like macOS or Windows, please read instructions specific for each project.
 Setup go environment:
 
 ```
-wget https://dl.google.com/go/go1.14.2.linux-amd64.tar.tz && tar xf go1.14.2.linux-amd64.tar.gz
+wget https://golang.org/dl/go1.14.2.linux-amd64.tar.gz && tar xf go1.14.2.linux-amd64.tar.gz
```
Comment (IKE471) [inline docs/build.md:197]: green

**Review by Lukasz257 — APPROVED** [review]

**Comment by Pymmdrza** [comment]
## The weakest and most problematic program that I have used so far and I regret it is this program that does not show any talent or diversity in it. address (different addresses) and finally, after spending a lot of time, if you don't have enough experience, you can't start this because there are thousands of obstacles in the way, so much so that some cases are poorly mentioned in it.
I now after 38 days when I wasted for this on 12 different servers I checked it now I removed it completely because it is very very weak and beginner and my advice to other friends to try other things with strong support More and more professional coding. Because you will face 100% problems in installation.

**Pull request merged**